app/assets/stylesheets/components/_foreword.scss

```diff
@@ -0,0 +1,167 @@
+$gutter: 40px;
+$trigger-width: 30px;
+$trigger-height: $trigger-width;
+
+.foreword {
+  padding: $gutter-large $gutter-small;
```

Comment:
Just looked into this and found that the `.foreword` class is a selector for the _individual foreword containers_ within the foreword section (see screenshot). I guess that's why the section-padding variable hasn't been used here.

Not sure about which solution to take here. The $section-padding-small variables could be a good solution. Add them into _variables.scss with the others..? I have a feeling we should avoid a) hard-coding the 32px value and b) referring to a variable like gutters which we may change later and accidentally affect parts of the page we didn't mean to. So a new variable seems like a good way to avoid both of those situations. What do you think @WKocur?

<img width="1419" alt="image" src="https://user-images.githubusercontent.com/56026099/120775645-507b9f80-c51b-11eb-9df3-ccdcf9748ae6.png">

Reply:
Hmmm it's a tricky one - the balance between forcing something that isn't the same to be the same, vs consistency! 

I think at this stage, don't worry about changing too much. The component is potentially sufficiently custom to argue that reusable variables aren't the way to g0 - the only thing I would do is not use the gutters for a top-bottom spacing value. 

I can't remember, but if the 32px is used in more than one place, then whack it in a variable, but it could perhaps stay in the foreword styles, if it's not used in another style file.

As a note, in general, gutters are pretty safe to use. They would be set at the beginning of a project and changing them would throw off all the designs for the designer, so they're unlikely to do it unless it's a full redesign/refresh.

Reply:
Okay, so where vertical spacing was done using a gutter variable, I am now just using 32px (that's about 4 places).

Where the gutter variable was used for horizontal spacing, I've left it in. Should all equate to the same spacing values, but now the gutter variable isn't being used for top-bottom spacing.

<img width="861" alt="image" src="https://user-images.githubusercontent.com/56026099/121008378-d6515200-c78a-11eb-9a93-0ce744d99e1b.png">